### ISSUE TITLE
Keeping Github Releases in sync with gem releases?

### ISSUE THREAD

**Issue by epugh** [omniauth/omniauth-oauth2#176]
I noticed that we have a 1.8.0 release on Rubygems, but the Releases in Github still reflects the older release.   

There are so many Rubygems out there that are somewhat abandoned, so I look at the github releases to get a sense of activity and maintenance state...   Would love as releases come out to keep both in sync.... 

Really appreciate this gem, it's been great for the http://github.com/o19s/quepid open source project!

**Comment by BobbyMcWho** [comment]
Sorry, I'm usually better about keeping up w/ creating the release from the tag, iirc, the changes were rather minor

**Comment by BobbyMcWho** [comment]
Actually @epugh I do have the 1.8.0 release on the releases tab, 1.7.3 is just a release I back created on the 1.7 line to not break some people's compatibility

**Comment by epugh** [comment]
You are quite right!   Maybe label the 1.8.0 the latest release (isn't that a checkbox that I've seen)???    And yeah, I am currently chasing the `omniauth-keycloak` gem to update so we can use the 1.8.0 version there!
